docs/admin/install/operator/configuring-serving-cr.md

```diff
@@ -299,10 +300,32 @@ spec:
       enabled: true
       knative-ingress-gateway:
         selector:
-          custom: ingressgateway
+          istio: ingressgateway
   config:
     istio:
-      gateway.knative-serving.knative-ingress-gateway: "custom-ingressgateway.istio-system.svc.cluster.local"
+      gateway.knative-serving.knative-ingress-gateway: "custom-ingressgateway.custom-ns.svc.cluster.local"
+```
+
+The key in `spec.config.istio` is in the format of `gateway.{{gateway_namespace}}.{{gateway_name}}`.
```

Comment:
```suggestion
The key in `spec.config.istio` is in the format of `gateway.<gateway-namespace>.<gateway-name>`.
```

Reply:
The style guide says to use `<placeholder-name>` for placeholders instead of `{{placeholder_name}}`